### DISCUSSION TITLE
Error: Cannot find module on first deployment

### DISCUSSION THREAD

**Discussion by bloodysneaker** [ZoeyVid/NPMplus#666] [Q&A] [answered]
Hi all,

I'm opening a discussion because I think that the issue is on me and not on NPM itself.
I've created the container but I encounter a "boot loop" when I try to start it.
I suspect that could be a low memory issue but I'd like a second opinion, I attach the log of one reitaration of the loop

[_nginx-proxy-manager_zoeyvid_logs.txt](https://github.com/ZoeyVid/NPMplus/files/14421157/_nginx-proxy-manager_zoeyvid_logs.txt)

Thanks in advance for your time
Alessio

**Comment by Zoey2936** [comment]
how did you installed npmplus?

**Reply by bloodysneaker** [reply]
Sorry my bad, deleted the sysinfo in the last review of my post.
Used the compose of the quickstart guide on an asustor  NAS with arm v8

**Reply by Zoey2936** [reply]
armv8 means arm64, right?

**Reply by bloodysneaker** [reply]
Yes, a realtek rtd1296 (1.4ghz quad core) if could be useful

**Reply by Zoey2936** [reply]
did you changed the working dir?

**Reply by bloodysneaker** [reply]
Nope, didn't make any changes.
In the meanwhile I've also tested the jc21/nginx-proxy-manager image and the ./logger error persist (the ./nginxbeautifier.js not, but could be not included in that image)  so maybe I've opened the discussion on the wrong repo.

**Reply by Zoey2936** [reply, marked as answer]
sorry, but I have no Idea
